app/models/art.rb

```diff
@@ -0,0 +1,9 @@
+class Art < ApplicationRecord
+  belongs_to :user
+
+  has_many_attached :photos
+
+  # validations
+  validates :title, :location, :description, :user, :category, :photos, presence: true
+  validates :price, presence: true, greater_than: 0
```

Comment:
:category needs to be removed

And height and width are not required but there should be a validation for integer and greater than 0